mackerel-plugin-accesslog/lib/accesslog.go

```diff
@@ -0,0 +1,183 @@
+package mpaccesslog
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/Songmu/axslogparser"
```

Comment:
axslogparser is so simple!

(I think some might want to specify the log format to parse, especially in Apache log format)